pinot-common/src/main/java/org/apache/pinot/common/utils/DataTable.java

```diff
@@ -30,22 +30,8 @@
  * Data table is used to transfer data from server to broker.
  */
 public interface DataTable {
+  // TODO: remove this when we stop supporting DataTable V2.
   String EXCEPTION_METADATA_KEY = "Exception";
-  String NUM_DOCS_SCANNED_METADATA_KEY = "numDocsScanned";
```

Comment:
Before removing these, please make sure and double-check that these are the exact strings (with no typos) used in `MetadataKeys`. I think some of these are returned in the results metadata by broker? We should take care not to break compat.

Reply:
I have made a double check. There is no typos.